src/amo/components/SearchSuggestion/index.js

```diff
@@ -1,22 +1,37 @@
 /* @flow */
 import * as React from 'react';
+import makeClassName from 'classnames';
 
 import Icon from 'ui/components/Icon';
 import LoadingText from 'ui/components/LoadingText';
 
 import './styles.scss';
 
 type Props = {|
-  name: string,
+  arrowAlt?: string,
   iconUrl: string,
   loading: boolean,
-  arrowAlt?: string,
+  name: string,
+  type: string,
 |};
 
-const SearchSuggestion = ({ name, iconUrl, arrowAlt, loading }: Props) => {
+const SearchSuggestion = ({
+  arrowAlt,
+  iconUrl,
+  loading,
+  name,
+  type,
+}: Props) => {
   return (
     <p className="SearchSuggestion">
-      <img alt={name} className="SearchSuggestion-icon" src={iconUrl} />
+      <img
+        alt={name}
+        className={makeClassName(
+          'SearchSuggestion-icon',
+          `SearchSuggestion-icon--${type}`,
```

Comment:
What if we add this class to the parent component? Like `SearchSuggestion--is-${type}`?